src/bashi/versions.py

```diff
@@ -123,6 +140,16 @@ def __str__(self) -> str:
 ]
 NVCC_GCC_MAX_VERSION.sort(reverse=True)
 
+
+# define the maximum supported gcc version for a specific cxx version
+# the latest supported cxx version must be added, even if the supported gcc version does not
+# increase
+GCC_CXX_SUPPORT: List[GccCxxSupport] = [
+    GccCxxSupport("14", "23"),
```

Comment:
I thought about this again. Actual the GCC 15 (not released yet) also adds missing C++23 and maybe the GCC 16 will too. So my idea is not good. Therefore we remove the entry for C++23 and replace it with C++17.
```suggestion
    GccCxxSupport("8", "17"),
```

The filter rule should still work. You need only to modify the tests. Sorry for the extra work.